src/gui/properties/proplistdelegate.cpp

```diff
@@ -156,3 +117,25 @@ void PropListDelegate::updateEditorGeometry(QWidget *editor, const QStyleOptionV
 {
     editor->setGeometry(option.rect);
 }
+
+void PropListDelegate::paint(QPainter *painter, const QStyleOptionViewItem &option, const QModelIndex &index) const
+{
+    switch (index.column())
+    {
+    case PropColumn::PROGRESS:
+        {
+            const int progress = static_cast<int>(index.data(TorrentContentModel::UnderlyingDataRole).toReal());
+            const int priority = index.sibling(index.row(), PropColumn::PRIORITY).data(TorrentContentModel::UnderlyingDataRole).toInt();
+            const bool isEnabled = static_cast<BitTorrent::DownloadPriority>(priority) != BitTorrent::DownloadPriority::Ignored;
```

Comment:
Wouldn't it be better if model provides `enabled` state. Or is it impossible?

Reply:
>Wouldn't it be better if model provides enabled state.

In general yes.

>Or is it impossible?

Currently impossible, it will require more refactor at the model as it uses additional user data role such as `TorrentContentModel::UnderlyingDataRole`.

Reply:
>Currently impossible, it will require more refactor at the model as it uses additional user data role such as `TorrentContentModel::UnderlyingDataRole`.

Isn't it enough to clear `Qt::ItemIsEnabled` flag of appropriate items?

Reply:
>Isn't it enough to clear Qt::ItemIsEnabled flag of appropriate items?

And how to clear it?

Reply:
Just don't add it here:
https://github.com/qbittorrent/qBittorrent/blob/a8ae97ba917259ec91868553aa881e4c0f0cd701/src/gui/torrentcontentmodel.cpp#L387

Reply:
The only question is whether it is really "disabled" in meaning, or whether we just draw it using the "disabled" color...

Reply:
>The only question is whether it is really "disabled" in meaning

It is really disabled, i.e. item cannot be selected. This is not we want.